docs/_includes/generated/execution_config_configuration.html

```diff
@@ -0,0 +1,122 @@
+<table class="table table-bordered">
```

Comment:
This config does not belong into the Flink core config documentation. But into a dedicated Table/Configuration page

Reply:
Agree. We should have a separate page, for example: `docs/dev/table/config.md` and add some more description on the top of the page.

Reply:
Thanks @twalthr  @wuchong , I have move this to a dedicated page.